src/content/guides/how-to-id-scam-tokens/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to identify scam tokens
+description: What are scam tokens, how do they make themselves look legitimate, and how to detect them to avoid the scam.
+lang: en
+---
+
+# How to identify scam tokens
```

Comment:
I'm noticing some of the other guides need these as well, but we should add custom header ids here (this is most important for translations)
```suggestion
# How to identify scam tokens {#how-to-identify-scam-tokens}
```